tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{36,37,38}-{offline,online,astropydev,numpydev,astropy31},build_docs,figure,conda
+envlist = py{36,37,38}-{offline,online,astropydev,numpydev,astropy31},build_docs,figure,figure_astropydev,conda
```

Comment:
Why this change?

Reply:
We forgot to add the new env to `tox.ini` in #3407